pkg/platform/abstract/project/external/leader/iguazio/types.go

```diff
@@ -0,0 +1,27 @@
+package iguazio
+
+const (
+	ProjectType = "project"
+)
+
+type Project struct {
+	Data ProjectData `json:"data,omitempty"`
+}
+
+type ProjectData struct {
+	Type string      `json:"type,omitempty"`
+	Attributes ProjectAttributes `json:"attributes,omitempty"`
+}
+
+type ProjectAttributes struct {
+	Name         string            `json:"name,omitempty"`
+	Namespace    string            `json:"namespace,omitempty"`
+	Labels       map[string]string `json:"labels,omitempty"`
+	Annotations  map[string]string `json:"annotations,omitempty"`
+	Description  string            `json:"description,omitempty"`
+	NuclioFields NuclioProject     `json:"nuclio_project,omitempty"`
```

Comment:
Field name, json name and its type doesnt seem to reflect what it is, perhaps a better name?

Reply:
Fixed name to be NuclioProject (forgot to fix it)
And this is how it's called on zebo - I'm only making it the same as it is there